python/ctranslate2/converters/opennmt_py.py

```diff
@@ -95,14 +98,23 @@ def _get_model_spec_seq2seq(
 def _get_model_spec_lm(opt, variables, src_vocabs, tgt_vocabs, num_source_embeddings):
     """Creates a model specification from the model options."""
     with_relative_position = getattr(opt, "max_relative_positions", 0) > 0
+    with_rotary = getattr(opt, "max_relative_positions", 0) == -1
+    with_alibi = getattr(opt, "max_relative_positions", 0) == -2
     activation_fn = getattr(opt, "pos_ffn_activation_fn", "relu")
     num_heads = getattr(opt, "heads", 8)
+    rotary_dim = 0 if with_rotary else None
+    ffn_glu = activation_fn == "silu"
 
     model_spec = transformer_spec.TransformerDecoderModelSpec.from_config(
         opt.dec_layers,
         num_heads,
         activation=_SUPPORTED_ACTIVATIONS[activation_fn],
+        ffn_glu=ffn_glu,
         with_relative_position=with_relative_position,
+        alibi=with_alibi,
+        rms_norm=opt.layer_norm == "rms",
+        rotary_dim=rotary_dim,
+        rotary_interleave=False,
```

Comment:
OpenNMT-py is using the definition from the original Llama code so it should be `True` (if the weights are not permuted).

```suggestion
        rotary_interleave=True,
```

`False` corresponds to the HF representation where the query/key are split in half: https://github.com/huggingface/transformers/blob/v4.29.1/src/transformers/models/llama/modeling_llama.py#L126